script/draft-release/run.ts

```diff
@@ -96,21 +111,81 @@ export async function run(args: ReadonlyArray<string>): Promise<void> {
   const noChangesFound = lines.every(l => l.trim().length === 0)
 
   if (noChangesFound) {
+    console.warn('No new changes found to add to the changelog. Aborting.')
     // print instructions with no changelog included
     printInstructions(nextVersion, [])
-  } else {
-    const changelogEntries = await convertToChangelogFormat(lines)
-
-    console.log("Here's what you should do next:\n")
-
-    if (channel === 'production') {
-      // make sure we only include entries since the latest production release
-      const existingChangelog = getChangelogEntriesSince(previousVersion)
-      const entries = [...existingChangelog]
-      printInstructions(nextVersion, entries)
-    } else if (channel === 'beta') {
-      const entries = [...changelogEntries]
-      printInstructions(nextVersion, entries)
+    return
+  }
+  console.log(`Setting app version to "${nextVersion}" in app/package.json...`)
+
+  try {
+    // this can throw
+    // sets the npm version in app/
+    execSync(`npm version ${nextVersion} --allow-same-version`, {
+      cwd: join(__dirname, '..', '..', 'app'),
+      encoding: 'utf8',
+    })
+    console.log(`Set!`)
+  } catch (e) {
+    console.warn(`Setting the app version failed 😿
+    (${e.message})
+    Please manually set it to ${nextVersion} in app/package.json.`)
+  }
+
+  const currentChangelog: IChangelog = require(changelogPath)
+  // if it's a new production release, make sure we only include
+  // entries since the latest production release
+  const newEntries =
+    channel === 'production'
+      ? [...getChangelogEntriesSince(previousVersion)]
+      : [...(await convertToChangelogFormat(lines))]
```

Comment:
Wondering if we want to explicitly check that the channel is `beta` here (like we did before these changes). Right now we're doing this anytime the channel is not `production` (so if it's undefined, or some other string, etc)

Reply:
good point. fixed in 6823bc683.